### PR TITLE
 Add banner actions block and flag to hide close button

### PR DESCRIPTION
**Pull request by dmitry-zaets**
This PR introduces:
- Fix https://github.com/segmentio/consent-manager/issues/104
- A possibility to add default buttons to accept/deny cookies via `bannerActionsBlock`

| Default Buttons | Without Close Button |
|---|---|
| image | image|
- A possibility to add custom buttons block via `bannerActionsBlock`

| Custom Buttons | Without Close Button |
|---|---|
| image|image |

Mobile:
| Default Buttons | Without Close Button | Custom Buttons | Without Close Button |
|---|---|---|---|
| image | image | image | image |

- Hide Clone button via `bannerHideCloseButton`
- Fix the dialog on iPhone

| Before | After |
|---|---|
| 4437EEAD-15AE-406D-9CEC-DC3CFF10CA22 | B117A412-708B-48B3-9161-87A0C445887E |

### CODE REVIEW

**Comment by dmitry-zaets** [comment]
Resolves https://github.com/segmentio/consent-manager/issues/138

File: README.md [hunk 1]
```diff
@@ -615,7 +634,7 @@ The CDN to fetch list of integrations from
 To run our storybook locally, simply do:
 
 ```
-$ yarn dev
+yarn dev
```
Comment (dmitry-zaets) [inline README.md:637]:
```suggestion
$ yarn dev
```

File: README.md [hunk 2]
```diff
@@ -625,8 +644,8 @@ and the storybook should be opened in your browser. We recommend adding a new st
 This package follows semantic versioning. To publish a new version:
 
 ```
-$ npm version <new-version>
-$ npm publish
+npm version <new-version>
+npm publish
```
Comment (dmitry-zaets) [inline README.md:648]:
```suggestion
$ npm version <new-version>
$ npm publish
```

File: src/consent-manager/container.tsx [hunk 3]
```diff
@@ -134,6 +137,7 @@ const Container: React.FC<ContainerProps> = props => {
     }
 
     if (props.closeBehavior === CloseBehavior.ACCEPT) {
+      props.setPreferences(props.preferences)
```
Comment (dmitry-zaets) [inline src/consent-manager/container.tsx:140]: Fix https://github.com/segmentio/consent-manager/issues/104

File: src/consent-manager/container.tsx [hunk 4]
```diff
@@ -128,23 +131,32 @@ const Container: React.FC<ContainerProps> = props => {
     }
   }, [isDialogOpen])
 
+  const onAcceptAll = () => {
+    props.setPreferences(props.preferences)
```
Comment (dmitry-zaets) [inline src/consent-manager/container.tsx:135]: Fix #104

**Comment by dmitry-zaets** [comment]
@pooyaj can you also check this or?

**Comment by felipe-najson-ntf** [comment]
Hello @dmitry-zaets, I hope you are well. I would like to know why you closed this pull-request.
I was reviewing it and it seems to me that it solves some of the issues that the library currently has and perhaps it could be merged into it.

Greetings, Segment Team.

**Comment by dmitry-zaets** [comment]
@felipe-najson-ntf ah, yes, we've also added a bunch of other features to our fork version. We can reopen a bigger PR if you would like, or we can do it step by step.

**Comment by felipe-najson** [comment]
@dmitry-zaets such good news! I will be working to resolve the library issues in these months. Maybe it's better to do it step by step, but if you want you can tell me about other improvements you have been working on and we can incorporate them, maybe with that you have already solved other issues in the library...

Regards, Felipe.

**Comment by felipe-najson-ntf** [comment]
@dmitry-zaets such good news! I will be working to resolve the library issues in these months. Maybe it's better to do it step by step, but if you want you can tell me about other improvements you have been working on and we can incorporate them, maybe with that you have already solved other issues in the library...

Regards, Felipe.

**Comment by dmitry-zaets** [comment]
Closing in favor of #164 164

**Pull request closed without merging**